microcosm_flask/formatting/csv_formatter.py

```diff
@@ -23,6 +26,17 @@ def build_headers(self, headers, **kwargs):
 
         return headers
 
+    def build_response(self, response_data):
+        response = Response(
+            self.format(response_data),
+            content_type=get_content_type(self.content_type, 'utf-8')
+        )
+
+        # start the output with U+FEFF BYTE ORDER MARK
+        # to signal to Excel to import the text file as UTF-8 rather than a legacy encoding
+        response.charset = "utf-8-sig"
```

Comment:
Should this move to a constant var? or is this value available from any of `microcosm_flask` dependencies 🤔

Reply:
It's unfortunately not a constant in the stdlib `codecs` or `encodings.utf_8_sig` - will add some constants here